postdeploy.sh

```diff
@@ -1,10 +1,5 @@
 #!/bin/bash
 
```

Comment:
Can we get some comments on how this works.  Why would $HEROKU_APP_NAME not be defined?
Why is the heroku command being run in a nested heroku command?

Reply:
Updated! Turned out that the nested command was unnecessary.